lib/commons/aria/get-role.js

```diff
@@ -1,10 +1,115 @@
 import getExplicitRole from './get-explicit-role';
 import getImplicitRole from './implicit-role';
+import lookupTable from './lookup-table';
+import isFocusable from '../dom/is-focusable';
 import { getNodeFromTree } from '../../core/utils';
 import AbstractVirtuaNode from '../../core/base/virtual-node/abstract-virtual-node';
 
+// when an element inherits the presentational role from a parent
+// is not defined in the spec, but through testing it seems to be
+// when a specific HTML parent relationship is required and that
+// parent has `role=presentation`, then the child inherits the
+// role (i.e. table, ul, dl). Further testing has shown that
+// intermediate elements (such as divs) break this chain only in
+// Chrome.
+//
+// Also, any nested structure chains reset the role (so two nested
+// lists with the topmost list role=none will not cause the nested
+// list to inherit the role=none).
+//
+// from Scott O'Hara:
+//
+// "the expectation for me, in standard html is that element
+// structures that require specific parent/child relationships,
+// if the parent is set to presentational that should set the
+// children to presentational.  ala, tables and lists."
+// "but outside of those specific constructs, i would not expect
+// role=presentation to do anything to child element roles"
+const inheritsPresentationChain = {
+	// valid parent elements, any other element will prevent any
+	// children from inheriting a presentational role from a valid
+	// ancestor
+	td: ['tr'],
+	th: ['tr'],
+	tr: ['thead', 'tbody', 'tfoot', 'table'],
+	thead: ['table'],
+	tbody: ['table'],
+	tfoot: ['table'],
+	table: [],
+	li: ['ol', 'ul'],
+	ol: [],
+	ul: [],
+	dt: ['dl'],
+	dd: ['dl'],
+	dl: []
```

Comment:
I just looked these up, as far as I can tell this stuff doesn't work with dl, which is good because otherwise we'd have to deal with div being allowed to group dd / dt elements.

```suggestion
```

Reply:
It doesn't work in Chrome but does in Firefox, Safari/VO, and IE11/JAWS:

![image](https://user-images.githubusercontent.com/2433219/85162234-3bc8cd80-b21e-11ea-8165-63f4409e24f7.png)

![image](https://user-images.githubusercontent.com/2433219/85162290-5dc25000-b21e-11ea-90bc-9cd167bd68ec.png)

Reply:
I had assumed `<div>` would stop the chain as we've seen in other things, but after testing it does in fact carry through the role inheritance so we'll need to add that.